srcs/app/http/RequestConfig.cpp

```diff
@@ -0,0 +1,107 @@
+#include <RequestConfig.hpp>
+#include <sys/types.h>
+
+RequestConfig::RequestConfig(const ServerConfig &server_config,
+		const std::string &request_path) :
+	location_(FindLocation_(server_config, request_path)),
+	common_(location_ ? location_->common : server_config.common),
+	path_(location_ ? &location_->path : NULL),
+	limit_except_(location_ ? &location_->limit_except : NULL),
+	root_(common_.root),
+	client_max_body_size_(common_.client_max_body_size),
+	autoindex_(common_.autoindex),
+	index_(common_.index),
+	error_pages_(common_.error_pages),
+	upload_store_(common_.upload_store),
+	return_status_(server_config.common.return_url.empty() ?
```

Comment:
Maybe we can do a new private function for these two ternaries?
Maybe not worth it...

Reply:
agree with you, will update it!